### ISSUE TITLE
Reconsider fallback font stack

### ISSUE THREAD

**Issue by makenosound** [icelab/berg#63]
It’s not really working for me. For the sans, I think `Gill Sans` has a little too much personality so it’s very recognisably different from `National`. I think it’d be better to find something more neutral, or at least a closer match.

And for the serif, the difference in shape is really strong — I think Georgia is probably closest to Tiempos in shape.

**Comment by andyicelab** [comment]
Happy to adjust these — totally threw them in there quickly.

**Comment by andyicelab** [comment]
Have done some fiddling around — I reckon `Geneva` and `Georgia` are the way to go. Sound okay to you @makenosound?

**Comment by makenosound** [comment]
Definitely 👍  for `Georgia`. My only problem with `Geneva` is that it’s so much larger than `National` is at theoretically the same `font-size` — wonder if there’s an alternative that more closely matches in size as well.

**Comment by andyicelab** [comment]
```css
.serif { font-family: Georgia, serif; }
.sans { font-family: "Trebuchet MS", "Helvetica Neue", Helvetica, sans-serif; }
```

Without doing some size adjustment stuff (which I reckon we probably _should_ do for that landing page `<h1>` to make it extra nice), `Trebuchet MS` seems a close-ish match.

For reference:

![berg-font-fallback](https://cloud.githubusercontent.com/assets/1016074/16069241/2ed6525a-3310-11e6-8f31-628f8bb83f12.jpg)

**Comment by makenosound** [comment]
Yep, that works OK for me. Trebuchet is kind of ugly when it’s that big, but I don’t mind it at smaller sizes, and so def. worth doing a little extra to give that main intro block some extra love.

**Comment by andyicelab** [comment]
Implemented in: https://github.com/icelab/berg-assets/commit/ddd63deb823e3c49e8526fa0b6e8e0bc3ffc2b5d

**Comment by makenosound** [comment]
💯
